cmd/skopeo/utils_test.go

```diff
@@ -45,6 +45,8 @@ func TestImageOptionsNewSystemContext(t *testing.T) {
 	opts := fakeImageOptions(t, "dest-", []string{}, []string{})
 	res, err := opts.newSystemContext()
 	require.NoError(t, err)
+
+	res.CompressionFormat = nil // reflect doesn't work well with functions
```

Comment:
I can only vaguely guess what this refers to. And if the comparison using the existent `assert.Equal` does not work for that value, I’d expect that assertion to be made in some alternate way before making it invisible to the existing `assert.Equal`.

(Applies to all instances; maybe there should be a shared helper, no idea.)

Reply:
now that we have a default in containers/image, there is no need of the special handling, going to drop these